std/dynamic_library.zig

```diff
@@ -19,6 +19,85 @@ pub const DynLib = switch (builtin.os) {
     else => void,
 };
 
+const LinkMap = extern struct {
+    l_addr: usize,
+    l_name: [*]u8,
+    l_ld: [*c]elf.Dyn,
+    l_next: [*c]LinkMap,
+    l_prev: [*c]LinkMap,
+
+    pub const Iterator = struct {
+        lm_ptr: [*c]LinkMap,
+
+        fn end(self: *const Iterator) bool {
```

Comment:
This can probably be simply: 

```suggestion
        fn end(self: Iterator) bool {
```